src/components/common/Button/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { ButtonHTMLAttributes, MouseEventHandler, ReactNode } from 'react';
+
+// Styles
+import './button.css';
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant?: string;
+  children?: ReactNode;
+  onClick?: MouseEventHandler<HTMLButtonElement>;
+}
+
+const Button = ({ variant = '', children = '', onClick }: ButtonProps) => {
+  let buttonClassName = '';
+
+  switch (variant) {
+  case 'primary':
+    buttonClassName = 'button-primary';
+    break;
+  case 'secondary':
+    buttonClassName = 'button-secondary';
+    break;
+  default:
+    buttonClassName = 'button-tertiary';
+  }
```

Comment:
You should not use like this.
we can use `className=`{button ${variant}}``

Reply:
Done